pf9/cluster/exceptions.py

```diff
@@ -31,7 +32,7 @@ def __init__(self, msg):
 
 
 class ClusterAttachFailed(ClusterCLIException):
-    def __init__(self, msg):
+    def __init__(self, msg,ctx,host):
```

Comment:
Why are we passing additional args here if we are not using them?

Reply:
We initially thought of uploading the logs even in cases where prep-node is successful but cluster-attach or created failed. These args were required to call the upload function from within the exception class.  Will be removing it now as we are only targetting prep-node failures.